crates/biome_js_formatter/src/utils/string_utils.rs

```diff
@@ -14,6 +14,8 @@ pub(crate) enum StringLiteralParentKind {
     Expression,
     /// Variant to track tokens that are inside a member
     Member,
+    /// Variant to track tokens that are inside an assert
+    Assert,
```

Comment:
Import assertion is the old name for import attributes. I think we should choose another variant name. Maybe:
```suggestion
    /// Variant to track tokens that are inside an import attribute
    ImportAttribute,
```